draft-ietf-ppm-dap.md

```diff
@@ -2632,52 +2573,171 @@ distinct aggregation parameters. If the batch has been queried with more than
 one distinct aggregation parameter, the Aggregator MUST abort with error of type
 "batchQueriedMultipleTimes".
 
-Finally, the Aggregator checks that the batch does not contain a report that was
-included in any previous batch. If this batch overlap check fails, then the
-Aggregator MUST abort with error of type "batchOverlap". For time-interval
-tasks, it is sufficient (but not necessary) to check that the batch interval
-does not overlap with the batch interval of any previous query. If this batch
-interval check fails, then the Aggregator MAY abort with error of type
-"batchOverlap".
+Next, the Aggregator checks if the set of batch bucket identified by by the
```

Comment:
```suggestion
Next, the Aggregator checks if the set of batch buckets identified by by the
```

(ooh, this is a very nice way to phrase this requirement now that we have the concept of "batch buckets" explicitly specified)